### PR TITLE
test: Add test for networking and cpu load/saturation

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

**Comment by marusak** [comment]
Has wrong Closes line as @martinpitt was a second faster when I was opening this PR :)

File: test/check-application [hunk 1]
```diff
@@ -155,6 +155,38 @@ class TestApplication(testlib.MachineCase):
         # Followed by virtually no data
         self.assertLessEqual(self.getMaximumSpike("network", False, 1598950800000, 6), 0.01)
 
+    def testCPU(self):
+        b = self.browser
+        m = self.machine
+
+        self.prepareArchive("cpu_network.tar.gz", 1598918400)
```
Comment (martinpitt) [inline test/check-application:162]: Can you please merge the tests, to avoid preparing a new VM for exactly the same setup?

**Review by martinpitt — CHANGES_REQUESTED** [review]
Nice, thanks for adding these!

File: test/check-application (hunk 1 above)
Comment (marusak) [reply test/check-application:162]: sure, so just one test like `testNetworkAndCpu`?

Comment (martinpitt) [reply test/check-application:162]: Yes, sounds fine -- for perf-graphs it still doesn't matter so much, but cockpit's tests run long enough as they are.

**Review by martinpitt — APPROVED** [review]
Thanks! Let's go with that for now.

**Pull request merged**